docs/usage/volume-populator/index.rst

```diff
@@ -0,0 +1,161 @@
+=======================================
+ReplicationDestination Volume Populator
+=======================================
+
+.. toctree::
+   :hidden:
+
+.. sidebar:: Contents
+
+   .. contents:: ReplicationDestination Volume Populator
+      :local:
+
+VolSync includes a Volume Populator controller to allow creation of PVCs that use a dataSourceRef that refers to a
+ReplicationDestination resource.
```

Comment:
I want to provide a short lead-in that conveys "why I should keep reading". This is the best I've come up with. Feel free to adjust or discard.

```suggestion
When a PVC is created that directly references a ReplicationDestination object, VolSync's Volume Populator controller will automatically fill the PVC with the most recent replicated data, alleviating the need to manually specify the name of the Snapshot.
```